### PR TITLE
Differentiation of Automate and Automate HA in CLI Page

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
### :nut_and_bolt: Description: What code changed, and why?

Automate HA customers were providing feedback/concerns for a long time on identifying HA-specific CLI commands vs Standalone Automate CLI commands. One of the biggest challenges was a technical limitation in the existing docs framework, limiting any better customer-friendly solution from being delivered.

### :chains: Related Resources

Epic Link: https://chefio.atlassian.net/browse/SHIELD-177

**Development Stories:**

- https://chefio.atlassian.net/browse/STALWART-306
- https://chefio.atlassian.net/browse/SHIELD-220
- https://chefio.atlassian.net/browse/STALWART-315
- https://chefio.atlassian.net/browse/STALWART-314
- https://chefio.atlassian.net/browse/STALWART-330 (Final Story)

**Previously Tried PRs:**

- https://github.com/chef/automate/pull/7614
- https://github.com/chef/automate/pull/7653
- https://github.com/chef/automate/pull/7660
- https://github.com/chef/automate/pull/7662

### :+1: Definition of Done

**POC**: https://chefio.atlassian.net/l/cp/cQv1DdKd
**List of CLIs for Standalone and HA**: https://chefio.atlassian.net/l/cp/4Ru3a7Gu (Refer to the page to review the CLI pages)

### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable

f9b9377e-9c2b-4c09-a40c-0fcfb21986da

### CODE REVIEW

File: components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands_ha.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
+{{ $data_path := index $.Site.Data (split (.Get "data_path") "/") }}
```
Comment (IanMadd) [inline components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands_ha.html:1]: You shouldn't need to duplicate this. You can just point the automate_cli_commands shortcode at a different data set.

File: components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands.html [hunk 2]
```diff
@@ -2,44 +2,101 @@
...
+         <td>
+            <div class="cell medium-4">{{ if .shorthand }}-{{ .shorthand }}, {{ end }}--{{ .name }}</div>
+         </td>
+         <td>
+            <div class="cell medium-8">{{ .usage }}{{ if .default_value }} (default: {{ .default_value }}){{ end }}</div>
+         </td>
+         {{ end }}
+      </tr>
+      {{- end -}}
+   </table>
```
Comment (IanMadd) [inline components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands.html:45]: This should be a description list. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl
For example, like this: https://docs.chef.io/workstation/ctl_chef/#options

File: components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands.html [hunk 3]
```diff
@@ -2,44 +2,101 @@
...
+   td:first-of-type{
+   width: 40%;
+   }
+   tr:nth-child(even) {
+   background-color: #fff !important;
+   }
+   .table-heading {
+   background: #eeeeee;
+   }
+</style>
```
Comment (IanMadd) [inline components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands.html:102]: You can delete this, we already have table styling.

File: components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands_ha.html [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+         <td>
+            <div class="cell medium-8">{{ .usage }}{{ if .default_value }} (default: {{ .default_value }}){{ end }}</div>
+         </td>
+         {{ end }}
+      </tr>
+      {{- end -}}
+   </table>
+</div>
+<h4>Supported On</h4>
+<p>{{ .supported_on }}</p>
```
Comment (IanMadd) [inline components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands_ha.html:48]: Something like this:
```suggestion
<p>This command is supported on the following node types: {{ .supported_on }}</p>
```

Comment (IanMadd) [reply components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands_ha.html:48]: You should probably add something where, if there aren't any types listed, it says "This command is supported on all node types."

Comment (dishanktiwari2501) [reply components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands_ha.html:48]: On the HA page, all the commands will be supported with one node. 
The UX team and the leads have decided on the header with the content.

File: components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands.html [hunk 5]
```diff
@@ -2,44 +2,101 @@
...
+         <td>
+            <div class="cell medium-3"><a href="#{{ $sa_command_id }}">{{ $sa_command }}</a></div>
+         </td>
+         <td>
+            <div class="cell medium-9">{{ $sa_string }}</div>
+         </td>
+      </tr>
+      {{ end }}
+   </div>
+</table>
```
Comment (IanMadd) [inline components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands.html:78]: This should just be a bulleted list without the command description.

**Review by kalroy — APPROVED** [review]

File: components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands_ha.html [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+   td:first-of-type{
+   width: 40%;
+   }
+   tr:nth-child(even) {
+   background-color: #fff !important;
+   }
+   .table-heading {
+   background: #eeeeee;
+   }
+</style>
```
Comment (IanMadd) [inline components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands_ha.html:104]: This has to go

```suggestion
```

File: components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands.html (hunk 3 above)
Comment (IanMadd) [inline components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands.html:102]: Delete this

```suggestion
```

**Review by IanMadd — APPROVED** [review]

**Pull request merged**